m.txt

```diff
@@ -0,0 +1,5 @@
+<<<<<<< HEAD
+most srufser
+=======
+more totally dif stuff
+>>>>>>> newbranch3
```

Comment:
comminting on multiple lines

Reply:
not sure why i wasn't able to 'request review' when submitting my comment it was a disabled option...

Reply:
replying to this thread in convo page